docs/paper-joss/paper.md

```diff
@@ -0,0 +1,134 @@
+Similarly the adjoint operator is a Type 1 NUFFT:
+
+: Correspondance Table between NUFFT and MRI acquisition model.
+
+| NUFFT Type | Operation | MRI Transform      | Operator               |
+|:-----------|:----------|:-------------------|:-----------------------|
+| Type 1     | Adjoint   | Kspace $\to$ Image | $\mathcal{F}_\Omega^*$ |
+| Type 2     | Forward   | Image $\to$ Kspace | $\mathcal{F}_\Omega$   |
+
+
+### Parallel Imaging Model
+In MRI the acquired signal can be received by multiple antennas (\"coils\"). 
+Each coil possesses a specific sensitivity profile (i.e. each sees the object differently due to its physical layout).
+
+$$\begin{aligned}
+\tilde{\boldsymbol{y}} = \begin{bmatrix}
+ \mathcal{F}_\Omega S_1 \\
+ \vdots  \\
+ \mathcal{F}_\Omega S_L \\
+ \end{bmatrix}
+ \boldsymbol{x} + \boldsymbol{n}_\ell  = \mathcal{F}_\Omega S \otimes \boldsymbol{x} + \tilde{\boldsymbol{n}}
+\end{aligned}$$
+
+Where $S_1, \dots, S_L$ are the sensitivity maps of each coil. 
+Such sensitivity maps can be acquired separetely by acquiring low frequency of the kspace, or estimated from the data.
+
+### Off-resonance correction model
+The constant magnetic field applied in a MRI machine $B0$ (with a typical intensity 1.5, 3 or 7 Tesla) is inherently disturbed at tissue interfaces with owing to different magnetic susceptibilities (such as air-tissue interfaces in the nose and ear canals). 
+Those field perturbations introduce a spatially varying phase shift in the frequencies acquired (noted $\Delta\omega_0$), making the acquisition model deviating from the convenient Fourier model. 
+Fortunately, this inhomogeneity map can be acquired separately or estimated and integrated in the model as:
+
+$$y(t_i) = \int_{\mathbb{R}^d} x(\boldsymbol{u}) e^{-2\imath\pi \boldsymbol{u} \cdot\boldsymbol{\nu_i} + \Delta\omega(\boldsymbol{u}) t_i} d\boldsymbol{u}$$
+
+where $t_i$ is the time at which the frequency $\nu_i$ is acquired.
+With these mixed-domain field pertubations, the Fourier model does not hold anymore and the FFT algorithm cannot be used any longer. 
+The main approach [@sutton_fast_2003] is to approximate the mixed-domain exponential term by splitting it into single-domain weights $b_{m, \ell}$ and $c_{\ell, n}, where $L \ll M, N$ regular Fourier transforms are performed to approximate the non-Fourier transform.
+
+$$x(\boldsymbol{u_n}) = \sum_{\ell=1}^L c_{\ell, n} \sum_{m}^M y(t_m) b_{m, \ell} e^{2\imath\pi \boldsymbol{u} \cdot \boldsymbol{\nu_i}}$$
+
+The coefficients $B=(b_{m, \ell}) \in \mathbb{C}^{M\times L}$ and $C=(c_\ell, n) \in \mathbb{C}^{L\times N}$ can be optimally estimated within MRI-NUFFT.
+
+## Trajectories generation and expansions 
+MRI-NUFFT comes with a wide variety of Non Cartesian trajectory generation routines, that have been gathered from the literature. It also provides ways of expanding existing trajectories. It is also able to export to specific formats, to be used in other toolboxes and on MRI hardware.
```

Comment:
Maybe we add links to docs? Is it allowed?

Reply:
No need the docs is reviewed along the paper itself. and the repo is linked in it already